src/button.js

```diff
@@ -2,11 +2,16 @@ import React, { Component } from 'react';
 
 class Button extends Component {
   render() {
+    const { id, state, toggleButton } = this.props;
+
     return (
-      <div className="Button">
+      <div
+        onClick={() => toggleButton(id)}
+        className={`Button ${ state === 'active' && 'Button--active'}`}>
```

Comment:
Another common pattern is using some props to interpolate strings like the `class` of an element.